README.md

```diff
@@ -7,3 +7,44 @@ Status:
 
 A plugin for [commitizen](https://github.com/commitizen/cz-cli) built for
 freeCodeCamp
+
+**Table of Contents**
+
+- [Introduction](#introduction)
+- [Local Installation](#local-installation)
+- [Usage](#usage)
+
+# Introduction
+
+This Commitizen adapter helps standardize commit messages for freeCodeCamp so
+being asked to fix incorrectly formatted commit messages will be less of a
+problem.
+
+# Local Installation
+
+```shell
+# Install general tool globally
+npm install -g commitizen
+
+# Install freeCodeCamp specific adapter
+git clone https://github.com/freeCodeCamp/cz-freecodecamp.git
+
+# Change accordingly to your path structure
+echo '{ "path": "cz-freecodecamp" }' > ~/.czrc
```

Comment:
This won't point to to the correct file. You need the whole file path.

My current czrc file looks like this:

```json
{ "path": "/usr/local/lib/node_modules/cz-conventional-changelog" }
```
It points to the path of my globally installed npm modules. We will need to update these instructions to including finding the path of the git clone and adding that the this echo command